docs/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Is this UserStory done?
+
+You might be confused about what it really means for a User Story to be 'done'.
+
+Well you're in luck! We at the **DoD - Department of Done** have comprised a helpful checklist to see if your code meets our one true definition of done.
+
+The User Story is done if **and only if**:
```

Comment:
Fancy, but according to me not necessary in the template. Add the section "Related issues" instead (as mentionned by saarofel).

Reply:
yea, I was going to remove it after the review, I was just in the mood for some dumb stuff